kcwitools/optimal_extract_utility.py

```diff
@@ -0,0 +1,128 @@
+from __future__ import print_function
+
+import numpy as np
+import matplotlib.pyplot as plt
+from numpy.polynomial.chebyshev import chebfit,chebval
+
+
+
+# Adopted from some online stuff that uses these for optimal extraction
+
+def wavelegth_fitter( fluxcube, varcube, aperture_mask, wavelength, order, aperture_indices, plot=False, verbose=False):
+
+	"""
+	Fit polynomials in the wavelength direction of each pixel in an aperture. 
```

Comment:
This sounds to me that we are fitting a spectrum to an aperture 
which would be odd.  But I'll read on.

Reply:
Renamed to spatial_profile